mono/mini/exceptions-amd64.c

```diff
@@ -885,7 +885,7 @@ mono_arch_exceptions_init (void)
 	}
 }
 
-#if defined(TARGET_WIN32) && !defined(DISABLE_JIT)
+#if defined(TARGET_WIN32) && !defined(DISABLE_JIT) && defined(MONO_ARCH_HAVE_UNWIND_TABLE)
```

Comment:
BTW, that define should be unified with MONO_ARCH_HAVE_CODE_CHUNK_TRACKING as they are both defined under the same set of conditions.